### PR TITLE
Enable API Security RC tests in nodejs

### PR DESCRIPTION
**Pull request by uurien**
## Motivation
RC and enable by default merged in nodejs, let's enable the tests

## Changes

Enable API Security RC tests.

## Workflow

1. ⚠️ Create your PR as draft ⚠️
2. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
3. Mark it as ready for review
    * Test logic is modified? -> Get a review from RFC owner. We're working on refining the `codeowners` file quickly.
    * Framework is modified, or non obvious usage of it -> get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

:rocket: Once your PR is reviewed, you can merge it!

🛟 [#apm-shared-testing](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X) 🛟

## Reviewer checklist

* [x] [Relevant label](https://github.com/DataDog/system-tests/blob/main/docs/CI/labels.md) (`run-parametric-scenario`, `run-profiling-scenario`...) are presents
* [x] No system-tests internal is modified. Otherwise, I have the approval from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)
* [ ] CI is green, or failing jobs are not related to this change (and you are 100% sure about this statement)
* [ ] A docker base image is modified?
    * [ ] the relevant `build-XXX-image` label is present
    * [ ] To R&P team: locally build and push the image to hub.docker.com 
* [ ] A scenario is added (or removed)?
    * [ ] Get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)
    * [ ] Once merged, add (or remove) it in system-test-dasboard nightly

### CODE REVIEW

File: manifests/nodejs.yml [hunk 1]
```diff
@@ -7,8 +7,8 @@ tests/:
   appsec/:
     api_security/:
       test_api_security_rc.py:
-        Test_API_Security_RC_ASM_DD_processors: missing_feature
-        Test_API_Security_RC_ASM_DD_scanners: missing_feature
+        Test_API_Security_RC_ASM_DD_processors: v5.3.0
+        Test_API_Security_RC_ASM_DD_scanners: v5.3.0
         Test_API_Security_RC_ASM_processor_overrides_and_custom_scanner: missing_feature # waf does not support it yet
       test_apisec_sampling.py:
         Test_API_Security_sampling: missing_feature
```
Comment (simon-id) [inline manifests/nodejs.yml:14]: do we not havr that feature ?

**Review by robertomonteromiguel — APPROVED** [review]

File: manifests/nodejs.yml (hunk 1 above)
Comment (uurien) [reply manifests/nodejs.yml]: yes, and it is an XPASS. enabled, let's see the results

Comment (simon-id) [reply manifests/nodejs.yml]: Did you run the system-tests manually on the version you put in the manifest ? the CI doesn't check old versions

Comment (uurien) [reply manifests/nodejs.yml]: I checked that the code was there, and now I run the test in my local, for all variants (including next)

**Review by simon-id — APPROVED** [review]

**Pull request merged**